### PR TITLE
add packages find options for data disc

### PR DESCRIPTION
**Pull request by emmalu**
Meant to address the install error:
Install Error

Following docs:
https://setuptools.pypa.io/en/latest/userguide/datafiles.html#include-package-data

### CODE REVIEW

File: stac_ipyleaflet/core.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import requests
 
 from ipyleaflet import Map, DrawControl, WidgetControl, TileLayer, Popup
-from stac_discovery.stac_widget import StacDiscoveryWidget
+from .stac_discovery.stac_widget import StacDiscoveryWidget
```
Comment (vincentsarago) [inline stac_ipyleaflet/core.py:9]: Note: I personally prefer using `absolute` path to make sure everything is well installed, but that's not mandatory

```python
from stac_ipyleaflet.stac_discovery.stac_widget import StacDiscoveryWidget
```

File: stac_ipyleaflet/stac_discovery/stac_widget.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from ipywidgets import VBox, Output, Layout, Text, ToggleButtons, Dropdown, DatePicker, HBox, Textarea, Checkbox
 import pandas as pd
-import ipyevents
+from importlib.resources import files
```
Comment (vincentsarago) [inline stac_ipyleaflet/stac_discovery/stac_widget.py:3]: `importlib.resources.files` was introduced in python 3.9

if you want the library to support python 3.8 you need to add a dependency like https://github.com/cogeotiff/rio-tiler/blob/dev/pyproject.toml#L34

and then do something like https://github.com/cogeotiff/rio-tiler/blob/dev/rio_tiler/colormap.py#L24-L28

File: stac_ipyleaflet/stac_discovery/stac_widget.py [hunk 3]
```diff
@@ -21,7 +21,7 @@ def template(self) -> VBox():
         padding = "0px 0px 0px 5px"
         style = {"description_width": "initial"}
 
-        nasa_cmr_path = Path("stac_ipyleaflet/stac_discovery/catalogs/nasa_maap_stac.tsv")
+        nasa_cmr_path = files('stac_ipyleaflet.stac_discovery').joinpath('catalogs').joinpath('nasa_maap_stac.tsv')
```
Comment (vincentsarago) [inline stac_ipyleaflet/stac_discovery/stac_widget.py:24]:
```suggestion
nasa_cmr_path = files("stac_ipyleaflet") / "stac_discovery" / "catalogs" / "nasa_maap_stac.tsv" 
```

**Review by vincentsarago — CHANGES_REQUESTED** [review]

File: stac_ipyleaflet/stac_discovery/stac_widget.py (hunk 2 above)
Comment (emmalu) [reply stac_ipyleaflet/stac_discovery/stac_widget.py:3]: Perhaps a question better for @abarciauskas-bgse or @wildintellect - do we need to support Python 3.8?

**Review by vincentsarago — APPROVED** [review]

File: stac_ipyleaflet/stac_discovery/stac_widget.py (hunk 2 above)
Comment (wildintellect) [reply stac_ipyleaflet/stac_discovery/stac_widget.py:3]: I think it's ok to put a min requirement of Python 3.9

**Pull request merged**